devel/hello-world.lua

```diff
@@ -32,6 +42,30 @@ function HelloWorldWindow:init()
             frame={w=10, h=5},
             frame_style=gui.INTERIOR_FRAME,
         },
+        widgets.Divider{
```

Comment:
please set the properties to make it a flat divider instead of a connected divider:
```lua
                frame={h=1},
                frame_style_l=false,
                frame_style_r=false,
```